source/motor.txt

```diff
@@ -17,15 +17,16 @@ Motor (Async Driver)
 Introduction
 ------------
 
-**Motor** is the recommended asynchronous Python driver for MongoDB.
-It is compatible with `Tornado <http://www.tornadoweb.org/>`__ and
-`asyncio <https://docs.python.org/3/library/asyncio.html>`__. We recommend
-that you use this driver if you need to run asynchronous operations. If
-you do not need to access MongoDB in a non-blocking manner or from
-co-routines, we recommend that you use the :doc:`PyMongo </pymongo>`
-driver instead.
-
-Follow the links below to learn more about how to use the Motor driver:
+You can use Motor, the official MongoDB driver for asynchronous Python 
```

Comment:
I don't remember where we landed with the named drivers, but I wonder if we can incorporate "Welcome to the documentation site for..." in order to conform with the rest.

Reply:
Addressed on this comment https://github.com/mongodb/docs-ecosystem/pull/788#discussion_r789171742